Nodejs/Tests/AnalysisTests/AnalysisFile.cs

```diff
@@ -131,6 +131,20 @@ static class Analysis {
             List<AnalysisFile> files = new List<AnalysisFile>();
             foreach (var file in Directory.GetFiles(directory, "*", SearchOption.AllDirectories)) {
```

Comment:
Ahh, I think I understand where you're coming from now...

It's not "automated testing" per se, but it is still important that we are testing the code that is in the product. Otherwise if we break something in the product related to this, we won't figure it out. The analysis limits are getting set from the following location.
https://github.com/Microsoft/nodejstools/blob/master/Nodejs/Tests/AnalysisDriver/AnalysisDriver.cs#L170

So we should provide an option to also run with medium analysis set (instead of the boolean flag we have right now to distinguish between low/high)